cmd/fortifyExecuteScan.go

```diff
@@ -777,10 +773,44 @@ func appendToOptions(config *fortifyExecuteScanOptions, options []string, t map[
 		if len(t["djangoTemplatDirs"]) > 0 {
 			options = append(options, "-django-template-dirs", t["djangoTemplatDirs"])
 		}
-		if len(t["pythonExcludes"]) > 0 {
-			options = append(options, "-exclude", t["pythonExcludes"])
-		}
-		return append(options, t["pythonIncludes"])
+
+	default:
+		return options
```

Comment:
Why is the default option without exclude and scr? Should src and exclude not alsways be part as it is not related to the build tool

Reply:
Ups, didn't see this comment before merging, sorry. It is excluded in order to not introduce a semantic change compared to before this PR.